proposed/http-message.md

```diff
@@ -399,18 +411,22 @@ interface MessageInterface
      * @param string $name Case-insensitive header field name.
      * @return string|null
      */
-    public function getHeader($name);
+    public function getHeaderLine($name);
 
     /**
-     * Retrieves a header by the given case-insensitive name as an array of strings.
+     * Retrieves all message headers.
```

Comment:
Maybe something more explicit to help people understand why this method exists: Retrieves all message headers as an array of header name to string values.